draft-ietf-quic-tls.md

```diff
@@ -552,7 +552,7 @@ older than 1.3 is negotiated.
 
 QUIC requires that the initial handshake packet from a client fit within the
 payload of a single packet.  The size limits on QUIC packets mean that a record
-containing a ClientHello needs to fit within 1171 octets.
+containing a ClientHello needs to fit within 1162 octets.
 
```

Comment:
That's not the only change required to the TLS document. The connection ID is also used as in the creation of Handshake Secrets. Pointing out the change in length may not be strictly necessary, but it will be helpful for implementers.